catalogd/cmd/catalogd/main.go

```diff
@@ -42,6 +42,7 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 	"k8s.io/klog/v2"
 	"k8s.io/klog/v2/textlogger"
+	"k8s.io/utils/ptr"
```

Comment:
Just a nit 

> UPSTREAM: <drop>: leaderelection settings

I think it need to have the PR from upstream ID

Reply:
We want this to be a <drop> commit. such that when upstream is finally able to be merged it, we don't have conflicts.

Reply:
https://github.com/operator-framework/operator-controller/pull/1663

Reply:
The `UPSTREAM: <pr>:` format is for backports where we've stopped automatic syncing (i.e. `release-4.x` branches)